src/main/scala/inox/Main.scala

```diff
@@ -50,7 +51,8 @@ trait MainHelpers {
       "Use solvers s1,s2,...\nAvailable: " +
       solvers.SolverFactory.solverNames.toSeq.sortBy(_._1).map {
         case (name, desc) => f"\n  $name%-14s : $desc"
-      }.mkString("")
+      }.mkString("") +
+      "\nYou can prefix the solvers unrollz3, smt-z3, smt-z3:<exec>, smt-cvc4, with 'noinc:' to use them in non-incremental mode"
```

Comment:
Isn't the prefix 'no-inc' (with a dash) ?

Reply:
Also, maybe make it `and smt-cvc4` and remove the comma after `smt-cvc4`?

Reply:
Good catch, thanks